src/App.vue

```diff
@@ -1,12 +1,21 @@
 <template>
   <div id="app">
     <router-view/>
+    <page-home/>
+    <app-modal/>
   </div>
 </template>
 
 <script>
+import AppModal from './components/AppModal'
+import PageHome from './components/PageHome'
+
 export default {
-  name: 'App'
+  name: 'App',
+  components: {
```

Comment:
dont declare components globally.

Reply:
As I understand these components is registered only for App component, not for its childs, is not it?